### PR TITLE
Refactoring custom label references in HEDA setting page

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

- We've changed the custom label references on the HEDA Settings page to access custom labels directly. Specifically, we've replaced all $Label.hed.(LabelName) with $Label.c.(LabelName). This has the added benefit of cleaning up our code base and ensuring consistency and maintainability.

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
1.  Set up a new dev org#1
2. Create a dummy package
3. Add all heda components to the dummy package
4. Set up another new dev org#2
5. Install the dummy package in dev org#2 
6. Make sure everything in HEDA setting page still working as before

### CODE REVIEW

**Comment by xinnanlisf** [comment]
@juliesnow this PR is ready for review. Thank you!

File: src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp [hunk 1]
```diff
@@ -31,50 +31,25 @@
...
-                    <aura:set attribute="else">
-                        <ui:outputURL aura:id="addrTabBtn" value="#" label="{!$Label.c.stgTabAccCont}" click="{!c.tabNavigationClick}" class="addrTab accs-cons-menu-item" />
-                    </aura:set>
-                </aura:if>
+                <ui:outputURL aura:id="addrTabBtn" value="#" label="{!$Label.c.stgTabAccCont}" click="{!c.tabNavigationClick}" class="addrTab accs-cons-menu-item" />
             </li>
             <li aura:id="systemTab" id="systemTab" class="slds-tabs__item slds-text-heading--label" title="System" role="tab">
                 <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
-                    <ui:outputURL aura:id="systemTabBtn" value="#" label="{!$Label.hed.stgTabSystem}" click="{!c.tabNavigationClick}" class="systemTab system-menu-item" />                    
+                    <ui:outputURL aura:id="systemTabBtn" value="#" label="{!$Label.hed.stgTabSystem}" click="{!c.tabNavigationClick}" class="systemTab system-menu-item" />
```
Comment (lparrott) [inline src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp]: @xinnanlisf Do we still need this, or can we just use the second link? 

It looks like a number of components still have 'hed__' references actually: 
- STG_CMP_Addr.cmp
- STG_CMP_Container.cmp
- STG_CMP_Rel.cmp
- STG_CMP_System.cmp
- STG_CMP_Tabs.cmp

Comment (xinnanlisf) [reply src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp]: Hey @lparrott, good catch! These should be good now.

**Comment by annejackson** [comment]
Tested heda settings page in dev org with dummy package, with a normal scratch org and with a namespaced scratch org. Labels and all looks good. Merging PR. Thanks @xinnanlisf

**Pull request merged**